ci/testnet-deploy.sh

```diff
@@ -64,10 +67,12 @@ Deploys a CD testnet
    -f                   - Discard validator nodes that didn't bootup successfully
    -w                   - Skip time-consuming "bells and whistles" that are
                           unnecessary for a high-node count demo testnet
-
+   -k                   - Amount to stake internal nodes.  If set, airdrops are disabled.
```

Comment:
Let's start using long opts for all new arguments  please.  The short args are basically unreadable (`-k`, `-K`) given that we've used almost every letter in the alphabet now

Reply:
Will do